internal/appsec/waf/waf_test.go

```diff
@@ -29,10 +29,11 @@ import (
 )
 
 func TestHealth(t *testing.T) {
-	version, err := Health()
-	require.NoError(t, err)
-	require.NotNil(t, version)
-	require.Equal(t, "1.2.1", version.String())
+	require.NoError(t, Health())
+}
+
+func TestVersion(t *testing.T) {
+	require.Equal(t, "1.3.0", Version())
```

Comment:
It's probably time to improve this test that has always been a pain to maintain and never was actually helpful:
```suggestion
	require.Regexp(t, `[0-9]+\.[0-9]+\.[0-9]+`, Version())
```